templates/web/common/includes/display_product_search_or_add.tt.html

```diff
@@ -0,0 +1,22 @@
+<!-- start templates/[% template.name %] -->
+
+[% IF !server_options_producers_platform %]
+  [% display_search_image_form %]
+[% END %]
+
+<form method="post" action="/cgi/product.pl" enctype="multipart/form-data">
+
+    <div class="row collapse">
+        <div class="small-9 columns">
+          <input type="text" name="code" placeholder="[% lang('or') %] [% lang('barcode') %]">
```

Comment:
lang('or') is not the same thing as what is in $template_data_ref_content->{or}

Reply:
Using $template_data_ref_content->{or}as [% or %] in template file was throwing an error. 
`"Unexpected token or [% or %]" `
So i used it directly, still thinking of way for 

```
$or =~ s/( |\&nbsp;)?://;
```